plinth/utils.py

```diff
@@ -63,3 +66,49 @@ def is_user_admin(request, cached=False):
     user_is_admin = request.user.groups.filter(name='admin').exists()
     request.session['cache_user_is_admin'] = user_is_admin
     return user_is_admin
+
+
+def get_domain_names():
```

Comment:
Perhaps we can replace the extra get_domain_names functions in matrix-synapse and diaspora modules.